apps/projects/app/components/Content/Settings.js

```diff
@@ -208,42 +210,47 @@ class Settings extends React.Component {
       </Button>
     )
 
+    const Separator = styled.hr`
```

Comment:
We decided at some point to put styled components at the end of the files as a convention, consider sticking to the convention.

Reply:
Sure!